tests/postgres_tests/test_aggregates.py

```diff
@@ -158,8 +154,9 @@ def test_ordering_warns_of_deprecation(self):
             self.assertEqual(values, {"arrayagg": [2, 1, 0, 0]})
         self.assertEqual(ctx.filename, __file__)
 
+    # RemovedInDjango61Warning: Remove this test
     def test_ordering_and_order_by_causes_error(self):
-        with self.assertWarns(RemovedInDjango61Warning):
+        with warnings.catch_warnings(record=True, action="always") as wm:
```

Comment:
Things got a little weird here when I update the warnings to use 7.0. The issue being that interacting with the class this way triggers both a 6.1 warning (for the `order_by` usage) and 7.0 warnings (for using the PostgreSQL StringAgg). I couldn't get `assertWarns` to work with both getting thrown so dug in and tried using the underlying behaviors of it to get it down.

Let me know if there is a better way to write this, and I will refactor it.

Reply:
I think the way you did it is great, it's a peculiar case given it triggers two distinct warnings of different categories 👍